spanner/google/cloud/spanner_v1/database.py

```diff
@@ -208,14 +209,11 @@ def create(self):
                 options=options,
             )
         except GaxError as exc:
-            if exc_to_code(exc.cause) == StatusCode.ALREADY_EXISTS:
-                raise Conflict(self.name)
-            elif exc_to_code(exc.cause) == StatusCode.NOT_FOUND:
-                raise NotFound('Instance not found: {name}'.format(
-                    name=self._instance.name,
-                ))
+            exception = exceptions.from_grpc_error(exc.cause)
+            if (exception.grpc_status_code == StatusCode.ALREADY_EXISTS
+                    or exception.grpc_status_code == StatusCode.NOT_FOUND):
+                raise exception
```

Comment:
1. I don't see any issue just raising it always (I realize the unit tests will have a branch for that solo `raise` below, but we can just update the unit tests).
1. You should use an `isinstance()` check there (rather than equality against `exception.grpc_status_code`)